appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/DraggableChildren.kt

```diff
@@ -97,80 +105,97 @@ fun <InteractionTarget : Any, ModelState : Any> DraggableAppyxComponent(
-                                        )
-                                    }
-                            )
-                            CompositionLocalProvider(
-                                LocalMotionProperties provides elementUiModel.motionProperties
-                            ) {
-                                element.invoke(
-                                    elementUiModel.copy(
-                                        modifier = Modifier
-                                            .then(elementUiModel.modifier)
-                                            .onPlaced {
-                                                size = it.size
-                                                val localCenter = Offset(
-                                                    it.size.width.toFloat(),
-                                                    it.size.height.toFloat()
-                                                ) / 2f
+                                                },
+                                            )
+                                        }
+                                        .offset { -elementOffset }
+                                        .then(elementUiModel.modifier)
+                                        .onPlaced {
+                                            elementSize = it.size
+                                            val localCenter = Offset(
+                                                it.size.width.toFloat(),
+                                                it.size.height.toFloat()
+                                            ) / 2f
 
-                                                transformedBoundingBox =
-                                                    it.boundsInParent()
-                                                        .inflate(gestureExtraTouchAreaPx)
-                                                offsetCenter =
-                                                    transformedBoundingBox.center - localCenter
-                                            }
-                                    )
+                                            transformedBoundingBox =
+                                                it.boundsInParent().inflate(gestureExtraTouchAreaPx)
+                                            offsetCenter =
+                                                transformedBoundingBox.center - localCenter
+                                        }
                                 )
-                            }
+                            )
                         }
                     }
                 }
+            }
         }
     }
 }
+
+
+@Composable
+fun elementOffset(
+    elementSize: IntSize,
+    containerSize: IntSize,
+): IntOffset {
+
+    val positionInside = motionPropertyRenderValue<Value, PositionInside>()
+    val positionOutside = motionPropertyRenderValue<PositionOutside.Value, PositionOutside>()
```

Comment:
What about importing `PositionInside.Value` and `PositionOutside.Value` using aliases to make this code more orthogonal? Something like this:
```kotlin
import com.bumble.appyx.interactions.core.ui.property.impl.position.PositionInside.Value as PositionInsideValue
import com.bumble.appyx.interactions.core.ui.property.impl.position.PositionOutside.Value as PositionOutsideValue
// ...
val positionInside = motionPropertyRenderValue<PositionInsideValue, PositionInside>()
val positionOutside = motionPropertyRenderValue<PositionOutsideValue, PositionOutside>()
```